### DISCUSSION TITLE
About the zip iso

### DISCUSSION THREAD

**Discussion by RelaxingNostalgia** [aldostools/webMAN-MOD#887] [General]
I haven't found any tutorial in the internet on how to use this experimental function,so how do I mount a zip compressed iso?

**Comment by aldostools** [comment]
You neet to compress a single ISO per zip. Put the zip in PSPISO and scan the game list.
When you mount a game in zip format, webMAN should execute PKGLAUNCH to decompress the zip in a temporary folder then mount the game.

This is an experimental feature, mainly for use with PSP games and may be broken since I haven't tested it in ages.

The same feature is implemented also in IRISMAN
